### PR TITLE
Added new sprites for the security techfab

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## New sprites for the Security TechFab

**Screenshots**
sectech

**Changelog**

- add: Added a new sprite for the Security TechFab. No more using old protolathe sprites!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -240,8 +240,21 @@
...
+    layers:
+    - state: icon
+      map: ["enum.ProtolatheVisualLayers.Base"]
+    - state: unlit
+      shader: unshaded
+      map: ["enum.ProtolatheVisualLayers.BaseUnlit"]
+    - state: icon
+      map: ["enum.ProtolatheVisualLayers.AnimationLayer"]
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:257]: There's only 3 states on the sprite but 4 layers? I imagine you might want to remove the icon above the unlit later.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml (hunk 1 above)
Comment (Elijahrane) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:257]: That's because the lathe visualizer breaks without it

**Comment by EmoGarbage404** [comment]
I closed this by accident while trying to fix something else like an idiot. MetalGearSloth, i'm unsure you still want me to edit that sprite layer after Rane's input.

**Comment by Elijahrane** [comment]
I'm planning on doing a lathe visualizer refactor some time here because now we have like 5 or 6 machines using it

**Comment by EmoGarbage404** [comment]
I'll be happy to contribute more general -fab based sprites for any refactoring in the future

**Pull request merged**